src/pages/About.jsx

```diff
@@ -33,9 +32,11 @@ const About = () => {
         Meet our team
       </h2>
 
+      {/* eslint-disable jsx-a11y/aria-role */}
```

Comment:
Remove these lint comments and rename 'role' to something else like 'group'

Reply:
That makes sense, thank you.